tests/installation/bootloader_settings/disable_boot_menu_timeout.pm

```diff
@@ -12,6 +12,7 @@ use base 'y2_installbase';
 
 sub run {
     $testapi::distri->get_installation_settings()->access_booting_options();
+    $testapi::distri->get_bootloader_settings()->switch_to_bootloader_options_navigating_tabs();
```

Comment:
Original code, tab to bootloader page and set grub timeout value in the same function 'sub disable_grub_timeout'.
I split this one function to two functions, one for switch to bootloader page, one for set grub timeout.
If we want to use right key to get bootloader page, we can reuse set grub timeout function.

Reply:
then we need tow method:
`switch_to_bootloader_options` (doesn't need any prefix, because it is the only way to navigate with libyui-rest-api, going directly).
`switch_to_bootloader_options_via_tabs`